contracts/types/LockData.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.20;
+
+import {LockDataLibrary} from "../libraries/LockDataLibrary.sol";
```

Comment:
keeping this in a separate file is breaking convention with types Currency + BalanceDelta. Wonder if it's worth migrating the library here

Reply:
moved it to types/LockData.sol to match